packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -384,10 +385,26 @@ export const start = async (zcf, privateArgs, baggage) => {
   /** @type {MapStore<Brand, Keyword>} */
   const brandToKeyword = provideDurableMapStore(baggage, 'brandToKeyword');
 
+  /** @type {MapStore<Brand, Subscriber<import('./auctionBook.js').BookDataNotification>>} */
+  const subscribers = provideDurableMapStore(baggage, 'subscribers');
+
   const reserveFunds = provideEmptySeat(zcf, baggage, 'collateral');
 
+  let bookCounter = 0;
+
   const makeAuctionBook = prepareAuctionBook(baggage, zcf);
 
+  const makeAuctionPublishKit = prepareDurablePublishKit(
+    baggage,
+    'Auction publish kit',
+  );
+  /** @type {PublishKit<import('./scheduler.js').ScheduleNotification>} */
+  const { publisher: schedulePublisher, subscriber: scheduleSubscriber } =
+    makeAuctionPublishKit();
+
+  const scheduleNode = E(privateArgs.storageNode).makeChildNode('schedule');
+  pipeTopicToStorage(scheduleSubscriber, scheduleNode, privateArgs.marshaller);
```

Comment:
if there's a risk that this PR lands after https://github.com/Agoric/agoric-sdk/issues/7300 is done then please include a TODO comment here to remove `pipeTopicToStorage`